### DISCUSSION TITLE
Add "loading" to "x-dropdown"

### DISCUSSION THREAD

**Discussion by LaMance22** [tasteui/tasteui#758] [Ideas]
Hi, it would be ideal if <x-dropdown> supported "loading". Is there a plan to implement such functionality in the future?

`<x-dropdown.items loading="myFunction" wire:click="myFunction" text="Name function" />`

**Comment by devajmeireles** [comment]
> Hi, it would be ideal if supported "loading". Is there a plan to implement such functionality in the future?
> 
> `<x-dropdown.items loading="myFunction" wire:click="myFunction" text="Name function" />`

Dropdown is a component to display static items. Why do you need a loading in it? I have no plans for that, but I would like to understand your idea.
